Source/ACE.Server/WorldObjects/WorldObject_Database.cs

```diff
@@ -18,7 +18,25 @@ partial class WorldObject
         /// This variable is set to true when a change is made, and set to false before a save is requested.<para />
         /// The primary use for this is to trigger save on add/modify/remove of properties.
         /// </summary>
-        public bool ChangesDetected { get; set; }
+        private bool changesDetected;
```

Comment:
This property shouldn't be checking DoNotSave

Reply:
Is there a better way to accomplish this? Why should it not check DoNotSave?

Reply:
You don't mess with ChangesDetected.

You manage your Save/Don'tSave in SaveBiota

Changes SHOULD still be detected even if DoNotSave is set. ChangesDetected has nothign to do with save conditions

Reply:
After looking back through it you are right. We can and should still detect changes. Removed this logic.